cli/command/formatter/stats.go

```diff
@@ -37,11 +35,80 @@ type ContainerStatsAttrs struct {
 	PidsCurrent      uint64 // Not used on Windows
 }
 
-// ContainerStats represents the containers statistics data.
+// ClearContainerStats represents the statistics data collected from a container.
+type ClearContainerStats struct {
```

Comment:
This name isn't good IMO. At first, it doesn't reflect what this structure does and then "clear containers" is intel containers technology :)